packages/client/src/pages/Forum/Forum.tsx

```diff
@@ -1,7 +1,293 @@
-const Forum = () => (
-  <>
-    <h1>Forum page</h1>
-  </>
-)
+import { RoutePath } from '@/core/Routes.enum'
+import { LikeOutlined } from '@ant-design/icons'
+import { Button, Input, Modal, Popover, Space, Table } from 'antd'
+import TextArea from 'antd/es/input/TextArea'
+import { ChangeEvent, useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+const Forum = () => {
+  const [isModalVisible, setIsModalVisible] = useState(false)
+  const [isCommentModalVisible, setIsCommentModalVisible] = useState(false)
+  const [newTopicName, setNewTopicName] = useState('')
+  const [newTopicContent, setNewTopicContent] = useState('')
+  const [newComment, setNewComment] = useState('')
+  const [dataSource, setDataSource] = useState<any[]>([])
+  const [selectedTopic, setSelectedTopic] = useState<any>(null)
+  const [selectedComments, setSelectedComments] = useState<any[]>([])
+
+  const navigate = useNavigate()
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = [
+        {
+          key: '1',
+          name: 'New games',
```

Comment:
_Не критично, но лучше исправить_

Для чистоты кода на этом этапе советую вынести моковые данные в отдельный файл для таких данных. Это приблизит код к его финальному состоянию и уберёт лишнее.
```suggestion
    const fetchData = async () => FORUM_DATA;
```